docs/exposition.rst

```diff
@@ -161,6 +161,29 @@ string::
 
 :ref:`Read more <set-fields>`
 
+Enum Fields
+-----------
+
+You can take advantage of MySQL's ``ENUM`` type through the ``EnumField``
+provided by django-mysql. Specifying the range of values available
+in an EnumField must be specified in the `choices` argument as a tuple
+or list::
+
+    class BookCover(Model):
+        color = EnumField(choices=['red', 'green', 'blue'])
+
+Like Django's standard `choices` argument, the values may be specified as a
+tuple specifying the actual value and a human readable name. The actual value
+is what will be stored in the database, while the human readable value will be shown
+in Django Admin forms where applicable::
+
+    class Book(Model):
+        color = EnumField(choices=[
+          ('red', 'Bright Red'),
+          ('green', 'Vibrant Green'),
+          'blue',  # human readable name will be set to "blue"
+        ])
+
```

Comment:
How would you like t-the link to be specified? Cant seem to find any other external references you used to mysql/mariadb in the exposition documentation